markdown/developers/tutorials/unofficial_pipelines.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Tutorial: Reusing nf-core components in non-nf-core pipelines. "
+subtitle: Guidance on how to use nf-core code and best practices in non-nf-core pipelines.
+---
+
+While we highly recommend and promote contributing to existing nf-core pipelines, in some cases you may disagree with some of the design decisions, or only wish to follow _parts_ of nf-core best practices for your own non-nf-core pipeline. This tutorial describes how to use nf-core code without making official nf-core pipelines.
+
+## Acknowledging nf-core
+
+If you use any form of nf-core code or infrastructure, we kindly ask that you acknowledge the nf-core initiative by adding the following markdown to your README or pipeline documentation.
+
+```markdown
+This pipeline uses code and infrastructure developed and maintained by the [nf-core](https://nf-co.re) initiative, and reused here under the [MIT license](https://github.com/nf-core/tools/blob/master/LICENSE).
+
+> The nf-core framework for community-curated bioinformatics pipelines.
+>
+> Philip Ewels, Alexander Peltzer, Sven Fillinger, Harshil Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso & Sven Nahnsen.
+>
+> Nat Biotechnol. 2020 Feb 13. doi: 10.1038/s41587-020-0439-x.
+>
+In addition, references of tools and data used in this pipeline are as follows:
+```
+
+## Removing nf-core only branding
+
+> Most of the following suggestions refer to when you have started your pipeline using the nf-core template.
+
+## General
+
+- Replace in all places reference to the pipeline name that includes the `nf-core/` suffix.
```

Comment:
```suggestion
- Replace all references to the pipeline name that include the `nf-core/` suffix.
```

Reply:
I think some linting tests will fail after doing this. Maybe worth giving an example lint config to get them to pass again.

As you said on slack - a global find and replace command would be good too.